Makefile

```diff
@@ -65,16 +55,31 @@ record:
 record-flex:
 	@go run src/dividat-driver/recorder/main.go ws://localhost:8382/flex
 
+.PHONY: bin
+bin:
+	mkdir -p bin
+
+
+VERSION := $(shell git describe --tags --abbrev=0)
```

Comment:
Can you explain the intent here? It doesn't seem desirable to me at first sight, don't we want to know the actual version, instead of treating any commit following the latest tag as the same version?

Reply:
Probably irrelevant, see discussion here: https://github.com/dividat/driver/pull/129#discussion_r1554962503